build/webpack/_development.js

```diff
@@ -32,8 +32,9 @@ export default (webpackConfig) => {
     // but _only_ when HMR is enabled. Putting this in the default development
     // configuration will break other tasks such as test:unit because Webpack
     // HMR is not enabled there, and these transforms require it.
+    let appliedHmr = false
     webpackConfig.module.loaders = webpackConfig.module.loaders.map(loader => {
-      if (/js(?!on)/.test(loader.test)) {
+      if (loader.loader === 'babel') {
```

Comment:
Since its a simple change, feel free to adjust on your next commit. I will close this PR.